src/include/common/copier_config/export_reader_config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <string>
+
+#include "csv_reader_config.h"
+#include "reader_config.h"
+
+namespace kuzu {
+namespace common {
+
+struct ExportReaderConfig {
```

Comment:
Let's not create an additional `ExportReaderConfig` and instead just reuse `ReaderConfig`.

The reason is `ExportReaderConfig` is exactly the same as `ReaderConfig` except that one keeps `csvOption` the other keeps `options`.

Note that `csvOption` can be derived from `option` through `CSVOption::construct`. So I don't see the difference between the two classes.

If you find this hard to do, then remove this class and open an issue for me.